code/datums/components/food_stuff.dm

```diff
@@ -5,6 +5,10 @@
 /// user = mob using the thing
 /// I = thing being eaten -- only does anything on components with a mob parent
 
+////// Argument formats! Any signals should really use these args in this order
+/// Eatsignal w/ mob parent  - M, user, I
+/// Eatsignal w/ item parent - I, M, user
```

Comment:
```suggestion
//--// Argument formats! Any signals should really use these args in this order
// Eatsignal w/ mob parent  - M, user, I
// Eatsignal w/ item parent - I, M, user
```

Reply:
One of these days I'll figure out when to use the right comment format!